src/components/program/data/hooks.jsx

```diff
@@ -22,7 +21,7 @@ export function useAllProgramData({ enterpriseUuid, programUuid }) {
       }
       return undefined;
     };
-    fetchData();
+    fetchProgramData();
```

Comment:
[question] Is the intent for this `useAllProgramData` hook to be removed, assuming its no longer used?

Reply:
Left it in at this point along with the `ProgramContextProvider` due the tests reliance on these components. Plan for refactoring it out once tests are rewritten.